configure

```diff
@@ -2187,15 +2187,36 @@ check_higgstools_libs() {
             return 1
         fi
     else
+        local filename
+        filename=$(basename -- "$found_lib")
+        local extension="${filename##*.}"
         local HIGGSTOOLSLIBS_="-lHiggsTools"
-        if [ -z "$higgstools_lib_dir" ]; then
-            # assume that the linker will look in the default paths
-            HIGGSTOOLSLIBS="$HIGGSTOOLSLIBS_"
-        else
-            # assume that the linker will look in the default paths
-            HIGGSTOOLSLIBS="-L$higgstools_lib_dir -Wl,-rpath,$higgstools_lib_dir $HIGGSTOOLSLIBS_"
-        fi
-        return 0
+        if [ "$extension" = "a" ] ; then
+            if [ -z "$higgstools_lib_dir" ]; then
+                # assume that the linker will look in the default paths
+                HIGGSTOOLSLIBS="-Wl,-Bstatic -$HIGGSTOOLSLIBS_ -Wl,-Bdynamic"
+            else
+                # assume that the linker will look in the default paths
+                HIGGSTOOLSLIBS="-L$higgstools_lib_dir -Wl,-rpath,$higgstools_lib_dir,-Bstatic $HIGGSTOOLSLIBS_ -Wl,-Bdynamic"
+            fi
+        elif [ "$extension" = "so" ] ; then
```

Comment:
Please also include `.dylib` here, which is the extension on Mac.

Reply:
Done in 60637b1.